target/linux/bcm53xx/image/Makefile

```diff
@@ -123,15 +123,15 @@ define Device/Default
   FILESYSTEMS := squashfs
   KERNEL_NAME := zImage
   IMAGE_NAME = $$(IMAGE_PREFIX)-$$(1).$$(2)
-  IMAGES := trx
+  IMAGES := factory.trx
```

Comment:
I actually wonder whether the "trx" is necessary as information at all here.
To me it looks like factory.bin would perfectly fit as well.

Reply:
There might be situations where two images would have the same purpose but different formats. Then this format identifier would be needed.
In general I think it is good to keep this kind of information.